manager/entrypoint.py

```diff
@@ -0,0 +1,82 @@
+import sys
+import os
+import datetime
+
+LIST_COMMAND = os.path.expandvars('aws s3 ls s3://$AWS_S3_BUCKET_NAME | grep -Eo "\d{2}-\d{2}-\d{4}"')
```

Comment:
Why not use boto3?

Reply:
I could honestly. If I was going to do anything other than use aws-cli I would probably just use TypeScript again, especially since the client will have already downloaded the image to run the other container. The problem came with recursively uploading a directory to S3. Turns out, its soooo much easier with the cli, otherwise I would have to write a filewalker to traverse the directory and upload each file one by one.